Decorator/TextDecorator/EmojiDecorator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace Decorator
+{
+    public class EmojiDecorator : TextFilter
```

Comment:
It should be only this decorator without others. But anyway I can't see it usage, please add.